### ISSUE TITLE
Model and Checkpoint files missing

### ISSUE THREAD

**Issue by swaticolab** [murat199/ViolanceDetection#1]
Hello Murat,
Thanks for providing this repository. Could you please provide me with the trained model and the checkpoint file.

**Comment by murat199** [comment]
Of course, I will upload it in one week. If Have you a question, you can ask to me. Don't hesitate. This project is violence detector as a real time. This project has two type. First type is using to pc camera (detect-master.py). First type is real time. Second type is using upload video with flask and html submit form (detect-upload.py).

I am using Flask-socket on the detect-master.
I am using Flask on the detect-upload.

**Comment by swaticolab** [comment]
Thanks Murat. I will surely let you know my queries, once I use the model.

**Comment by murat199** [comment]
You can reach to 'checkpoints' folder then this link;
https://1drv.ms/f/s!AmZhoCxvcn7ZablJKCH1c1nNwco
and
You can reach to darknet file then this link;
https://1drv.ms/u/s!AmZhoCxvcn7Zbh9uJMsxHNFQw4o
